Remove uses of sets() which make iteration order unpredictable. (#566)
SFDO-Tooling/Snowfakery: docs/salesforce.md

...
```

> The order in which sobjects is loaded is inferred by Snowfakery and CumulusCI

Reviewer: ```suggestion
The order in which sObjects are loaded is inferred by Snowfakery and CumulusCI
```

Reviewer 2: ```suggestion
Snowfakery and CumulusCI infer the order in which sObjects are loaded
```
